app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -40,15 +40,21 @@ class Identification extends React.Component {
       });
 
       // lodash's 'remove()' actually removes the values from the object
-      let supportedLangCodes = languageData.filter(l => l.supported.sms || l.supported.email || l.supported.phone).map(x => x.code);
-      const supportedLangsFormatted = _.remove(langOptions, n => supportedLangCodes.includes(n.value));
+      const fullySupportedLangCodes = languageData.filter(l => l.supported.sms && l.supported.email && l.supported.phone).map(x => x.code);
+      const supportedLangCodes = languageData.filter(l => l.supported.sms || l.supported.email || l.supported.phone).map(x => x.code);
+      const fullySupportedLangsFormatted = _.remove(langOptions, n => fullySupportedLangCodes.includes(n.value));
+      const partiallySupportedLangsFormatted = _.remove(langOptions, n => supportedLangCodes.includes(n.value));
       const unsupportedLangsFormatted = langOptions;
 
       // For each language, get the support options
       const groupedOptions = [
         {
-          label: 'Supported Languages',
-          options: supportedLangsFormatted,
+          label: 'Fully Supported Languages',
```

Comment:
thanks for doing this! just a thought, do we think we need the * anymore? they seem to clutter the list now that it is organized by fully/partial/unsupported and the warning gives more info about the partial and unsupported

Reply:
@hackrm thoughts?

Reply:
@sgober that sounds like a good idea to me. let's see what @hackrm and @tstrass think

Reply:
Yeah I would be fine with removing the asterisk, since it's now redundant. I could also see wanting to keep it since users may have already internalized what it means, and it provides extra emphasis that those languages are not fully supported, so I am fine either way and would just defer to @hackrm.

Reply:
I just preemptively removed it in this commit, if the PO team rules against it we can revert the change: https://github.com/SaraAlert/SaraAlert/pull/1062/commits/cfd0038ef467c57eaf3edaf4c54f0d726c33074c

Reply:
I think this is a lot cleaner, thanks @timwongj!